client/components/Navbar.tsx

```
@@ -109,97 +260,385 @@ const Navbar = () => {
+                                )}
+                              </div>
+                            )}
+                          </div>
+                          <div className="flex justify-between items-center mt-2">
+                            {result.basePrice && (
+                              <div className="text-green-700 font-medium">
+                                ${result.basePrice}/hr
+                              </div>
+                            )}
+                            <Link href={`/venues/${result._id}`} passHref>
+                              <button className="text-xs flex items-center text-blue-600 hover:text-blue-800">
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)